### PR TITLE
Added Motion/ACTIVITY_RECOGNITION permission

### LINKED ISSUE #73: Add missing activity recognition permissions for Android and iOS
Hi. Can you add missing Android and iOS permissions, please?

Android:
- ACTIVITY_RECOGNITIONHi. Can you add permissions mission Android and iOS permissions, please?

Android:

ACTIVITY_RECOGNITION

iOS:

MotionActivityManagerAuthorized

iOS:
- MotionActivityManagerAuthorized

### PR DESCRIPTION
**Pull request by charlee-dev**
Closes #73

### CODE REVIEW

File: permissions/src/iosMain/kotlin/dev/icerock/moko/permissions/ios/MotionPermissionDelegate.kt [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+import platform.CoreMotion.CMAuthorizationStatusDenied
+import platform.CoreMotion.CMAuthorizationStatusNotDetermined
+import platform.CoreMotion.CMAuthorizationStatusRestricted
+import platform.CoreMotion.CMMotionActivityManager
+import platform.Foundation.NSOperationQueue
+
+internal class MotionPermissionDelegate : PermissionDelegate {
+    override suspend fun providePermission() {
+        val manager = CMMotionActivityManager()
+        manager.startActivityUpdatesToQueue(NSOperationQueue.mainQueue) {}
```
Comment (Alex009) [inline permissions/src/iosMain/kotlin/dev/icerock/moko/permissions/ios/MotionPermissionDelegate.kt:14]: is it tested? we can request permission only by try to start updated receiving?

File: permissions/src/iosMain/kotlin/dev/icerock/moko/permissions/ios/MotionPermissionDelegate.kt [hunk 2]
```diff
@@ -0,0 +1,30 @@
+package dev.icerock.moko.permissions.ios
```
Comment (Alex009) [inline permissions/src/iosMain/kotlin/dev/icerock/moko/permissions/ios/MotionPermissionDelegate.kt:1]: add copyright like in other files please

**Review by Alex009 — CHANGES_REQUESTED** [review]

File: permissions/src/iosMain/kotlin/dev/icerock/moko/permissions/ios/MotionPermissionDelegate.kt (hunk 2 above)
Comment (charlee-dev) [reply permissions/src/iosMain/kotlin/dev/icerock/moko/permissions/ios/MotionPermissionDelegate.kt:5]: Added copyright.

File: permissions/src/iosMain/kotlin/dev/icerock/moko/permissions/ios/MotionPermissionDelegate.kt (hunk 1 above)
Comment (charlee-dev) [reply permissions/src/iosMain/kotlin/dev/icerock/moko/permissions/ios/MotionPermissionDelegate.kt]: I simplified providing the motion permission on iOS. This is the official way to request motion permission on iOS. We request it by when we use it for the first time, so we perform a single collection.

**Pull request merged**